server/models/shift.model.js

```diff
@@ -23,6 +23,11 @@ const shiftSchema = new theSchema({
     organization:
     {
         type: mongoose.Schema.Types.ObjectId
+    },
+    verified:
+    {
+        type: Boolean,
+        required: true
```

Comment:
For `verified` I believe if you don't specify a default value for a boolean attribute in Mongoose, it sets it to 'null'. I am not 100% sure though (you can probably just test it), but to be safe , set default value of verified to false.

Reply:
I did set it to false upon creation within the org route endpoint, is that good or should I change the schema too?

Reply:
That works but let's still have the default value set to false to be safe.